@stellar/anchor-tests/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@stellar/anchor-tests",
+  "version": "0.1.0",
+  "description": "stellar-anchor-tests is a library and command line interface for testing Stellar anchors.",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "./lib"
+  ],
+  "bin": {
+    "stellar-anchor-tests": "lib/cli.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stellar/stellar-anchor-tests.git"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "tsc",
+    "build:docs": "typedoc",
+    "prepare": "husky install"
+  },
+  "homepage": "https://github.com/stellar/stellar-anchor-tests",
```

Comment:
Lets change the home page here to point to the `@stellar/anchor-tests` directory so when we publish the package users can find the relevant README in one click.